src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -51,27 +51,15 @@
     <Error Condition="'$(PublishTrimmed)' == 'false'" Text="PublishTrimmed is implied by native compilation and cannot be disabled." />
 
     <!-- Fail with descriptive error message for common unsupported cases. -->
-<<<<<<< HEAD
     <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(OS)' == 'Windows_NT' and !$(RuntimeIdentifier.StartsWith('win')) and !$(RuntimeIdentifier.EndsWith('wasm'))"
```

Comment:
I think it would be best to set `DisableUnsupportedError` to `true` for WASM somewhere above to avoid future conflicts in this section.

Reply:
Added near the top, thanks